source/Octopus.Tentacle/Kubernetes/KubernetesConfig.cs

```diff
@@ -21,6 +21,7 @@ public static class KubernetesConfig
 
         public static TimeSpan PodsConsideredOrphanedAfterTimeSpan => TimeSpan.FromMinutes(int.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__PODSCONSIDEREDORPHANEDAFTERMINUTES"), out var podsConsideredOrphanedAfterTimeSpan) ? podsConsideredOrphanedAfterTimeSpan : 10);
         public static bool DisableAutomaticPodCleanup => bool.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__DISABLEAUTOPODCLEANUP"), out var disableAutoCleanup) && disableAutoCleanup;
+        public static bool DisablePodEventsInTaskLog => bool.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__DISABLEPODEVENTSINTASKLOG"), out var disable) && disable;
```

Comment:
Are we able to (and is it worth) feature flagging this somehow in the Octopus Server side instead? That way we can turn it off if we find there's an issue? I only worry since if something goes wrong here, there's very little chance of recovery, since we rely on Tentacle to run a deployment to update itself.

Reply:
We _could_, but that's a lot more work as we'd need to update the contracts + add UI to set/handle it.

Reply:
Probably not worth it for this change since it's fairly trivial then, but it may be worth considering this in future.